docs/cfp/cntug-cfp-en.md

```diff
@@ -0,0 +1,102 @@
+
+In order to promote more vendors or users to participate, Clound Native Taiwan User Gorup allows Sales or Marketing Pitches to be placed on the last few pages of each session. The number of pages will depend on the length of the presentation, up to a maximum of 2 pages, not including the opening and closing.
+
+## Submission Types
+
+| Type | Duration (mins) | # of Speaker | # of Marketing Pitches Pages|
+|------|:---------------:|:------------:|:---------------------------:|
+| Formal Presentation | 30 | 1 | 1 |
+| Solo Presentation | 60 | 1 | 2 |
+| Dual Presentation | 60 | 2 | 2 |
+| Panel Discussion | 60 | 3 - 5 | Depneds on # of speakers |
+| Lighting Talk | 15 | 1 | 0 |
+
+## Meetup Information
+
+- Date: `Depends on hoster and speaker availability`
+- Timezone: `GMT+0800`
+- Time: `19:00 - 22:00`
+- Location: `in-person` or `online`
+- Max. no of seats: `Depends on venue`
+- Registration Website: [CNCF Community Group - Cloud Native Taiwan User Group][4]
+- Language: Prefer `Mandarin Chinese`, but `English` is also welcome
+- Video recorded on YouTube: The lecturer is expected to consent to being recorded on video but will request permission from the lecturer before sharing the recording on [CNTUG YouTube][5]. Alternatively, the lecturer may have already indicated that they will not record the meeting.
+
+### Online Meetup Tools
+
+Choose one of the following:
+
+- Microsoft Teams: Ability to provide real-time captioning
+- Google Meet: Generic Selection
+- YouTube Live Stream: Anyone can see it without logging in
+
+### In-person Meetup Location
+
+All reservations must be made one month in advance.
+
+Choose one of the following:
+
+| Location | # of Seats | Fee |
+|:--------:|:----------:|:---:|
+| [Tenlong 天瓏書局][6] | 40 | 100 NTD per seats |
+
+## Important Notes
+
+- All speakers are required to adhere to [CNCF Code of Conduct][2]
+- `Avoid sales or marketing pitches and discussing unlicensed or potentially closed-source technologies` when preparing your proposal; these talks are almost always rejected due to the fact that they take away from the integrity of our events, and are rarely well-received by conference attendees.
+
+## CODE OF CONDUCT
+
+CNCF and its project communities are dedicated to providing a harassment-free experience for participants at our events. All submitters are required to abide by our [Code of Conduct][2]
+
+## CFP Questions
+
+If you have any questions regarding the CFP process, please contact us at [organizers@cloudnative.tw](mailto:organizers@cloudnative.tw)
+
+[1]: https://landscape.cncf.io/
+[2]: https://github.com/cncf/foundation/blob/main/code-of-conduct.md
+[3]: https://github.com/cncf/communitygroups
+[4]: https://community.cncf.io/cloud-native-taiwan-user-group/
+[5]: https://www.youtube.com/channel/UCoYY8K9fbfDtTY7m68UCATA
+[6]: https://www.tenlong.com.tw/
+[7]: https://docs.google.com/forms/d/e/1FAIpQLSd67skDYwAnDl_MO5sWIb5YxvP8yYmV0Swo4oNu7LSVtgrsUA/viewform?pli=1
```

Comment:
It's better to use inline links instead of marking after hand IMO.
`[title](links)`

Reply:
How to do?

Reply:
But I think it's cleaner to use [][], that's how I've been using it so far.

Reply:
Nah, in this case the user will have to scroll to the bottom to click on the link and then scroll back again to continue. If you use inline links the user will only need to click on the link text to open the link.

Reply:
![image](https://github.com/cloud-native-taiwan/cloud-native-taiwan.github.io/assets/5253671/88e17014-4883-46fe-9d68-79e4a944fe81)

Markdown support inline and reference both methods

https://daringfireball.net/projects/markdown/syntax#link